2013/08/24 13:13:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "layers/view".' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/view')
#1 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/layers/3
HTTP_REFERER=http://lutskhistory.dev/index.php/layers/create
---
2013/08/24 14:42:30 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\Lutskhistory.dev\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('getPoints')
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/getPoint...')
#9 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/layers/getPoints?id=1
---
2013/08/24 14:55:48 [error] [php] Invalid argument supplied for foreach() (Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:781)
Stack trace:
#0 unknown(0): LayersController->actionGetPoints()
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): LayersController->runAction()
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): LayersController->filterAccessControl()
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): LayersController->runActionWithFilters()
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): LayersController->run()
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 Z:\home\Lutskhistory.dev\www\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/layers/getPoints?id=1
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (113)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 14:55:53 [error] [php] Invalid argument supplied for foreach() (Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:781)
Stack trace:
#0 unknown(0): LayersController->actionGetPoints()
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): LayersController->runAction()
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): LayersController->filterAccessControl()
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): LayersController->runActionWithFilters()
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): LayersController->run()
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 Z:\home\Lutskhistory.dev\www\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/layers/getPoints?id=1
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (113)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 14:56:24 [error] [php] Invalid argument supplied for foreach() (Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:781)
Stack trace:
#0 unknown(0): LayersController->actionGetPoints()
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): LayersController->runAction()
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): LayersController->filterAccessControl()
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): LayersController->runActionWithFilters()
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): LayersController->run()
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 Z:\home\Lutskhistory.dev\www\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/layers/getPoints?id=1
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (113)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 14:56:29 [error] [php] Invalid argument supplied for foreach() (Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:781)
Stack trace:
#0 unknown(0): LayersController->actionGetPoints()
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): LayersController->runAction()
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): LayersController->filterAccessControl()
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): LayersController->runActionWithFilters()
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): LayersController->run()
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 Z:\home\Lutskhistory.dev\www\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/layers/getPoints?id=1
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (113)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 14:56:34 [error] [php] Invalid argument supplied for foreach() (Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:781)
Stack trace:
#0 unknown(0): LayersController->actionGetPoints()
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): LayersController->runAction()
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): LayersController->filterAccessControl()
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): LayersController->runActionWithFilters()
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): LayersController->run()
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 Z:\home\Lutskhistory.dev\www\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/layers/getPoints?id=1
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (113)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 14:56:37 [error] [php] Invalid argument supplied for foreach() (Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:781)
Stack trace:
#0 unknown(0): LayersController->actionGetPoints()
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): LayersController->runAction()
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): LayersController->filterAccessControl()
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): LayersController->runActionWithFilters()
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): LayersController->run()
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 Z:\home\Lutskhistory.dev\www\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/layers/getPoints?id=1
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (113)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 14:56:44 [error] [php] Invalid argument supplied for foreach() (Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:781)
Stack trace:
#0 unknown(0): LayersController->actionGetPoints()
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): LayersController->runAction()
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): LayersController->filterAccessControl()
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): LayersController->runActionWithFilters()
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): LayersController->run()
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 Z:\home\Lutskhistory.dev\www\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/layers/getPoints?id=1
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (113)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 14:57:07 [error] [php] Invalid argument supplied for foreach() (Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:781)
Stack trace:
#0 unknown(0): LayersController->actionGetPoints()
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): LayersController->runAction()
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): LayersController->filterAccessControl()
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): LayersController->runActionWithFilters()
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): LayersController->run()
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 Z:\home\Lutskhistory.dev\www\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/layers/getPoints?id=1
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (113)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 15:04:40 [error] [php] Invalid argument supplied for foreach() (Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:781)
Stack trace:
#0 unknown(0): LayersController->actionGetPoints()
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): LayersController->runAction()
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): LayersController->filterAccessControl()
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): LayersController->runActionWithFilters()
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): LayersController->run()
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 Z:\home\Lutskhistory.dev\www\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/layers/getPoints?id=1
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (113)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 15:05:51 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Array' in 'where clause'. The SQL statement executed was: SELECT *
FROM `points`
WHERE `id` IN (Array).
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (114)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 15:05:51 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Array' in 'where clause'. The SQL statement executed was: SELECT *
FROM `points`
WHERE `id` IN (Array)' in Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:541
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php(395): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 Z:\home\Lutskhistory.dev\www\protected\models\Layers.php(114): CDbCommand->queryAll()
#2 Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php(188): Layers->getPoints('1')
#3 [internal function]: LayersController->actionGetPoints('1')
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(LayersController), Array)
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(LayersController), Object(ReflectionMethod), Array)
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('getPoints')
#15 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/getPoint...')
#16 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#17 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php/layers/getPoints?id=1
---
2013/08/24 15:10:21 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Array' in 'where clause'. The SQL statement executed was: SELECT *
FROM `points`
WHERE `id` IN (Array).
in Z:\home\Lutskhistory.dev\www\protected\models\Layers.php (118)
in Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php (188)
in Z:\home\Lutskhistory.dev\www\index.php (13)
2013/08/24 15:10:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Array' in 'where clause'. The SQL statement executed was: SELECT *
FROM `points`
WHERE `id` IN (Array)' in Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php:541
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\db\CDbCommand.php(395): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 Z:\home\Lutskhistory.dev\www\protected\models\Layers.php(118): CDbCommand->queryAll()
#2 Z:\home\Lutskhistory.dev\www\protected\controllers\LayersController.php(188): Layers->getPoints('1')
#3 [internal function]: LayersController->actionGetPoints('1')
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(LayersController), Array)
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(LayersController), Object(ReflectionMethod), Array)
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#13 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('getPoints')
#15 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/getPoint...')
#16 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#17 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php/layers/getPoints?id=1
---
2013/09/12 20:46:36 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php:336
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('GetPoints')
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/GetPoint...')
#10 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php/layers/GetPoints
HTTP_REFERER=http://lutskhistory.dev/maket/
---
2013/09/12 20:48:14 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php:336
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('GetPoints')
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/GetPoint...')
#10 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php/layers/GetPoints
HTTP_REFERER=http://lutskhistory.dev/maket/
---
2013/09/12 20:49:52 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php:336
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('GetPoints')
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/GetPoint...')
#10 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php/layers/GetPoints
HTTP_REFERER=http://lutskhistory.dev/maket/
---
2013/09/12 20:51:44 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php:336
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('GetPoints')
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/GetPoint...')
#10 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php/layers/GetPoints
HTTP_REFERER=http://lutskhistory.dev/maket/
---
2013/09/12 20:52:14 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php:336
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('GetPoints')
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/GetPoint...')
#10 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php/layers/GetPoints
HTTP_REFERER=http://lutskhistory.dev/maket/
---
2013/09/12 20:54:43 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php:336
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('GetPoints')
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/GetPoint...')
#10 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php/layers/GetPoints
HTTP_REFERER=http://lutskhistory.dev/maket/
---
2013/09/12 20:55:43 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php:336
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('GetPoints')
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('layers/GetPoint...')
#10 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php/layers/GetPoints
---
2013/09/12 22:19:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "getPointPosts".' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php:483
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(270): CController->missingAction('getPointPosts')
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('getPointPosts')
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('posts/getPointP...')
#3 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/posts/getPointPosts
---
2013/09/12 22:20:06 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php:336
Stack trace:
#0 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 Z:\home\Lutskhistory.dev\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(291): CFilterChain->run()
#7 Z:\home\Lutskhistory.dev\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(282): CController->run('getPointPosts')
#9 Z:\home\Lutskhistory.dev\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('points/getPoint...')
#10 Z:\home\Lutskhistory.dev\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 Z:\home\Lutskhistory.dev\www\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php/points/getPointPosts
---
